packages/web-components/src/global/variables.css

```diff
@@ -290,7 +289,6 @@
   --ic-z-index-navigation-menu: calc(var(--ic-z-index-base-value) + 60);
   --ic-z-index-side-navigation: calc(var(--ic-z-index-base-value) + 60);
   --ic-z-index-dialog: calc(var(--ic-z-index-base-value) + 100);
-  --ic-z-index-toast: calc(var(--ic-z-index-base-value) + 110);
```

Comment:
I think this needs to stay as it was just added in to fix z-index issues with toasts on the guidance site